cplusplus.json

```diff
@@ -25,5 +25,119 @@
+  {"id": "cpp.vcflib", "name": "vcflib", "tech": ["parsing"], "tags": ["vcf"], "repo": "vcflib/vcflib",
+  "imports": ["Variant.h"], "examples": ["https://github.com/vcflib/vcflib/tree/master/test/tests"]},
+  {"id": "cpp.wham", "name": "wham", "tech": ["computational-science"], "tags": ["genetics", "analysis",], "repo": "zeeev/wham",
+  "imports": ["api/BamMultiReader.h", "readPileUp.h"], "examples": []},
+
+  {"id": "cpp.bittorrent-dht", "name": "BitTorrent DHT", "tech": ["internet"], "tags": ["BitTorrent"], "repo": "jech/dht",
+  "imports": ["dht.h"], "examples": ["https://github.com/jech/dht/blob/master/dht-example.c"]},
+  {"id": "cpp.libtorrent", "name": "libtorrent", "tech": ["internet"], "tags": ["BitTorrent"], "repo": "arvidn/libtorrent",
+  "imports": ["libtorrent/entry.hpp", "libtorrent/bencode.hpp", "libtorrent/bdecode.hpp"], "examples": ["https://github.com/arvidn/libtorrent/blob/master/examples"]},
+  {"id": "cpp.libutp", "name": "libutp", "tech": ["internet"], "tags": ["BitTorrent"], "repo": "bittorrent/libutp",
+  "imports": ["utp.h"], "examples": ["https://github.com/bittorrent/libutp/blob/master/ucat.c"]},
+
+  {"id": "cpp.argh", "name": "Argh!", "tech": ["utils"], "tags": ["argument-parsing"], "repo": "adishavit/argh",
+  "imports": ["argh.h"], "examples": ["https://github.com/adishavit/argh/blob/master/example.cpp"]},
+  {"id": "cpp.args", "name": "args", "tech": ["utils"], "tags": ["argument-parsing"], "repo": "taywee/args",
+  "imports": ["args.hxx"], "examples": ["https://github.com/Taywee/args/tree/master/examples"]},
+  {"id": "cpp.boost-program_options", "name": "Boost.Program_options",  "tech": ["utils"], "tags": ["argument-parsing"], "repo": "boostorg/program_options",
+  "imports": ["boost/program_options.hpp"], "examples": ["https://github.com/boostorg/program_options/tree/develop/example"]},
+  {"id": "cpp.cli11", "name": "CLI11", "tech": ["utils"], "tags": ["argument-parsing"], "repo": "CLIUtils/CLI11",
+  "imports": ["CLI/CLI.hpp"], "examples": ["https://github.com/CLIUtils/CLI11/tree/master/examples"]},
+  {"id": "cpp.cxxopts", "name": "cxxopts", "tech": ["utils"], "tags": ["argument-parsing"], "repo": "jarro2783/cxxopts",
+  "imports": ["cxxopts.hpp"], "examples": ["https://github.com/jarro2783/cxxopts/tree/master/src"]},
+  {"id": "cpp.docopt.cpp", "name": "docopt.cpp",  "tech": ["utils"], "tags": ["argument-parsing"], "repo": "docopt/docopt.cpp",
+  "imports": ["docopt.h"], "examples": ["https://github.com/docopt/docopt.cpp/tree/master/examples"]},
+  {"id": "cpp.gflags", "name": "gflags", "tech": ["utils"], "tags": ["argument-parsing", "cflags"], "repo": "gflags/gflags",
+  "imports": ["gflags/gflags.h"], "examples": ["https://gflags.github.io/gflags/"]},
+  {"id": "cpp.ncurses", "name": "Ncurses", "tech": ["api"], "tags": ["gui", "framework"], "repo": "http://invisible-island.net/ncurses/",
```

Comment:
tech == gui, tags == framework

In general, please avoid using tech == "api" unless you have (like it's an API for integration with yelp or something) because it groups unrelated things into one category.